showcase/app/routes/page-components/advanced-table.ts

```diff
@@ -24,149 +28,27 @@ export const clone = <T>(arr: T[]): T[] => {
-    'badge-type': HdsBadgeTypes;
-    'badge-color': {
-      name: HdsBadgeColors;
-      key: number;
-    };
-    color: HdsBadgeColors;
-  };
-}
-
-interface User {
-  id: number;
-  name: string;
-  email: string;
-  role: 'Owner' | 'Admin' | 'Contributor';
-  isSelected?: boolean;
-  isAnimated?: boolean;
-}
-
-interface Cluster {
-  id: number;
-  'peer-name': string;
-  'cluster-partition': string;
-  status: string;
-  services: {
-    imported: number;
-    exported: number;
-  };
-}
-
-interface SelectableItem {
-  id: number;
-  lorem: string;
-  ipsum: string;
-  dolor: string;
-  isSelected: boolean;
-}
-
-interface UserWithMoreColumns {
-  first_name: string;
-  last_name: string;
-  email: string;
-  phone: string;
-  age: string;
-  education: string;
-  occupation: string;
-  bio: string;
-}
-
-interface SpanningEntity {
-  id: number;
-  name?:
-    | {
-        text: string;
-        rowspan?: number;
-      }
-    | string;
-  service?:
-    | {
-        text: string;
-        colspan?: number;
-      }
-    | string;
-  description?: string;
-  email?: string;
-}
-
-interface Policy {
-  id: number;
-  name: string;
-  description: string;
-  status: string;
-  isOpen?: boolean;
-  children?: Policy[];
-}
-
-interface PolicyCustom {
-  id: number;
-  name: string;
-  description: string;
-  status: string;
-  isOpen?: boolean;
-  data?: Policy[];
-}
-
 export default class PageComponentsAdvancedTableRoute extends Route {
-  async model() {
-    const responseMusic = await fetch('/api/folk.json');
-    const responseUserData = await fetch('/api/mock-users.json');
-    const responseClusters = await fetch('/api/mock-clusters-with-status.json');
-    const responseSelectableData = await fetch(
-      '/api/mock-selectable-data.json',
-    );
-    const responseManyColumns = await fetch('/api/mock-many-columns.json');
-    const responseSpanningManual = await fetch(
-      '/api/mock-spanning-cells-manual.json',
-    );
-    const responseNested = await fetch('/api/mock-nested-rows.json');
-    const responseNestedCustom = await fetch(
-      '/api/mock-nested-rows-custom.json',
-    );
-
-    const { data: music } = (await responseMusic.json()) as Record<
-      'data',
-      MusicEntity[]
-    >;
-    const userData = (await responseUserData.json()) as User[];
-    const clusters = (await responseClusters.json()) as Cluster[];
-    const selectableData =
-      (await responseSelectableData.json()) as SelectableItem[];
-    const manyColumns =
-      (await responseManyColumns.json()) as UserWithMoreColumns[];
-    const spanningManualData =
-      (await responseSpanningManual.json()) as SpanningEntity[];
-    const nestedData = (await responseNested.json()) as Policy[];
-    const nestedDataCustom =
-      (await responseNestedCustom.json()) as PolicyCustom[];
-
+  model() {
     return {
-      music: music.map((record) => ({ id: record.id, ...record.attributes })),
+      music: folkMusic.map((record) => ({
```

Comment:
you could potantially flattn this in the mock file it self, I understand that original json was copied from some sort of ember tutorial which showed how JSONAPI data would look like but we want flat structure so you don't have to do this manipulation

Reply:
Ah, I think that would be good to do. Will update.